PyPDF2/_cmap.py

```diff
@@ -284,7 +284,10 @@ def compute_space_width(
             w1[-1] = cast(float, ft1["/DW"])
         except Exception:
             w1[-1] = 1000.0
-        w = list(ft1["/W"])  # type: ignore
+        try:
+            w = list(ft1["/W"])  # type: ignore
+        except Exception:
+            w = []
```

Comment:
Would the following be equivalent?

```suggestion
        if "/W" in ft1:
            w = list(ft1["/W"])  # type: ignore
        else:
            w = []
```

Reply:
@MartinThoma,
sound good. any reason to prefer (for my improvement 😊)

Reply:
However, I don't have strong feelings about this one. If you have a reason for your preference, I will accept & merge that.